src/components/TransactionsList.js

```diff
@@ -1,35 +1,55 @@
 import React from 'react'
+import Transaction from './Transaction'
 
-const TransactionsList = () => {
+const renderTableHeaders = () => {
+  return(
+    <tr>
+      <th>
+        <h3 className="ui center aligned header">
+          Posted At
+        </h3>
+      </th>
+      <th>
+        <h3 className="ui center aligned header">
+          Description
+        </h3>
+      </th>
+      <th>
+        <h3 className="ui center aligned header">
+          Category
+        </h3>
+      </th>
+      <th>
+        <h3 className="ui center aligned header">
+          Amount
+        </h3>
+      </th>
+    </tr>
+  )
+}
 
+const renderTransactions = (props) => {
```

Comment:
I think you asked about this in lecture, but I prefer to keep this inside the component. You may even consider making this class based so that you can use helper methods.